testing-common/src/main/java/io/opentelemetry/instrumentation/testing/junit/db/SemconvStabilityUtil.java

```diff
@@ -39,14 +38,13 @@ private static Map<AttributeKey<?>, AttributeKey<?>> buildMap() {
     map.put(DB_SQL_TABLE, DB_COLLECTION_NAME);
     map.put(DB_CASSANDRA_TABLE, DB_COLLECTION_NAME);
     map.put(DB_MONGODB_COLLECTION, DB_COLLECTION_NAME);
-    map.put(DB_COSMOSDB_CONTAINER, DB_COLLECTION_NAME);
```

Comment:
was this removal intentional?

Reply:
oh yeah, I had meant to leave a github comment about it, it's unused (we don't have any cosmosdb tests)